spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -52,6 +52,8 @@
   it_behaves_like 'redundant', '(true)', 'true', 'a literal'
   it_behaves_like 'redundant', '(false)', 'false', 'a literal'
   it_behaves_like 'redundant', '(/regexp/)', '/regexp/', 'a literal'
+  it_behaves_like 'plausible', '("x"; "y")'
```

Comment:
>don't you need an assignment operator preceding those expressions to be sure the parens are needed?

Yes, however, I need to consider other cases as well.

```ruby
x = (
  foo
  bar
)


x + (
  foo
  bar
)


x((
  foo
  bar
))
```

If the parens is removed, the above code changes its meaning(and syntax error).
However, RuboCop adds offences to the code currently.



I'll update the cop. Thank you for your comment!